Demo/Demo/SwiftUIComponents/PayPalWeb/PayPalWebButtonsView.swift

```diff
@@ -1,24 +1,23 @@
 import SwiftUI
 import PaymentButtons
 
-struct PayPalTransactionView: View {
+struct PayPalWebButtonsView: View {
```

Comment:
Renamed from "TransactionView" to "ButtonView" since this contains all of the payment buttons and does not do anything with transactions